hooks/useRailgunTx.tsx

```diff
@@ -10,23 +10,29 @@ import {
 import {
   RailgunERC20AmountRecipient,
   RailgunERC20Amount,
-  NetworkName,
   TransactionGasDetailsSerialized,
   EVMGasType,
   NETWORK_CONFIG,
+  NetworkName,
   deserializeTransaction,
 } from "@railgun-community/shared-models";
 import { ethers } from "ethers";
-import { getAddress, parseUnits } from "ethers/lib/utils.js";
+import { parseUnits } from "ethers/lib/utils.js";
```

Comment:
For some reason VSCode always autocompletes the import to this JS file, but typically it's this
```suggestion
import { parseUnits } from "@ethersproject/units";
```

Reply:
0c23838